pkg/types/ovirt/config.go

```diff
@@ -0,0 +1,76 @@
+package ovirt
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+
+	"gopkg.in/yaml.v2"
+)
+
+var defaultOvirtConfigEnvVar = "OVIRT_CONFIG"
+var defaultOvirtConfigPath = filepath.Join(os.Getenv("HOME"), ".ovirt", "ovirt-config.yaml")
+
+// ErrCanNotLoadOvirtConfig
+var ErrCanNotLoadOvirtConfig error = errors.New("can not load ovirt config")
+
+// OvirtConfig holds oVirt api access details
+type OvirtConfig struct {
+	URL      string `yaml:"ovirt_url"`
```

Comment:
I *think* this should just be username & password while URL & CAFile(?) should still be in the install config. This is based on a few ideas:

- a user might have access to multiple URLs, so they just need edit the install-config and leave ovirt-config.yaml alone
- install-config (& metadata) might be shared between users who have different usernames/pw but use the same URL